pygmsh/surface_base.py

```diff
@@ -4,11 +4,13 @@
 
 class SurfaceBase(object):
     _ID = 0
+    num_edges = 0
 
-    def __init__(self, id0=None):
+    def __init__(self, num_edges=0, id0=None):
```

Comment:
> I had them initially in the reverse order, but that would imply that, in order to specify a name for the surface, it would be required to set also `num_edges`

Well, you can do anything when using the key, which I'd do anyhow. (`SurfaceBase(id0='lol', num_edges=10)` is more readable than `SurfaceBase('lol', 10)`.) In any case, if the signature if the signature is `__init__(self, id0=None, num_edges=0)` then `SurfaceBase('lol')` does what it has done so far: Setting the `id0`. Let's do that.